src/mongocrypt-marking.c

```diff
@@ -728,13 +728,33 @@ static bool _mongocrypt_fle2_placeholder_common(_mongocrypt_key_broker_t *kb,
     return false;
 }
 
+static bool _fle2_choose_contention_factor(mongocrypt_t *crypt,
+                                           int64_t exclusive_upper_bound,
+                                           int64_t *out,
+                                           mongocrypt_status_t *status) {
+    if (crypt->opts.contention_factor_fn) {
```

Comment:
```suggestion
                                           mongocrypt_status_t *status) {
    BSON_ASSERT_PARAM(crypt);
    BSON_ASSERT_PARAM(out);
    if (crypt->opts.contention_factor_fn) {
```
Assert preconditions.